backend/de.metas.business/src/main/java/de/metas/inventory/IInventoryDAO.java

```diff
@@ -49,5 +53,9 @@ default List<I_M_InventoryLine> retrieveLinesForInventoryId(final InventoryId in
 
 	void setInventoryLinesProcessed(InventoryId inventoryId, boolean processed);
 
+	void setInventoryLinesCounted(@NonNull InventoryId inventoryId, boolean counted);
```

Comment:
do u have a case when counted=false?
if not, pls rename the method to markInventoryLinesAsCounted and remove counted param